src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -115,8 +115,20 @@ function iterator(array $values): IteratorArgument
 
 /**
  * Creates a lazy iterator by tag name.
+ *
+ * @deprecated since version 4.4, to be removed in 5.0. Use "tagged_iterator" instead.
```

Comment:
```suggestion
 * @deprecated since Symfony 4.4, to be removed in 5.0. Use "tagged_iterator" instead.
```

Reply:
@nicolas-grekas Like that?

Reply:
Yes, also let's make this a single sentence: ", use"
You're right about the doc, there is a PR open to fix it.